src/importers/postman/Parser.js

```diff
@@ -0,0 +1,583 @@
+            return auth
+                .set(
+                    'key',
+                    this._referenceEnvironmentVariable(
+                        helper.accessKey
+                    )
+                )
+                .set(
+                    'secret',
+                    this._referenceEnvironmentVariable(
+                        helper.secretKey
+                    )
+                )
+                .set(
+                    'region',
+                    this._referenceEnvironmentVariable(
+                        helper.region
+                    )
+                )
+                .set(
+                    'service',
+                    this._referenceEnvironmentVariable(
+                        helper.service
+                    )
+                )
+        }
+    }
+
+    _extractHawkAuth(params, helper) {
+        let auth = new HawkAuth()
+
+        if (helper) {
+            return auth
+            .set(
+                'algorithm',
+                this._referenceEnvironmentVariable(
+                    helper.algorithm
+                )
+            )
+            .set(
+                'key',
+                this._referenceEnvironmentVariable(
+                    helper.hawk_key
+                )
+            )
+            .set(
+                'id',
+                this._referenceEnvironmentVariable(
+                    helper.hawk_id
+                )
+            )
+        }
+    }
+
+    _extractOAuth1(params) {
+        let auth = new OAuth1Auth()
+        if (!params) {
+            return auth
+        }
+
+        const paramMap = {
+            oauth_consumer_key: 'consumerKey',
+            oauth_signature_method: 'algorithm',
+            oauth_timestamp: 'timestamp',
+            oauth_nonce: 'nonce',
+            oauth_version: 'version',
+            oauth_signature: 'signature'
+        }
+
+        let kvList = (params || '').split(',')
+        for (let kvStr of kvList) {
+            let [ key, value ] = kvStr.split('=')
+            if (paramMap[key]) {
+                auth = auth.set(
+                    paramMap[key],
+                    this._referenceEnvironmentVariable(
+                        value.replace(/(^")|("$)/g, '')
+                    )
+                )
+            }
+        }
+        return auth
+    }
+
+    _extractAuth(authLine, helperType, helper) {
+        let [ , scheme, params ] = authLine.match(/([^\s]+)\s(.*)/)
+
+        let helperMap = {
+            basicAuth: ::this._extractBasicAuth,
+            digestAuth: ::this._extractDigestAuth,
+            awsSigV4: ::this._extractAWSS4Auth,
+            hawkAuth: ::this._extractHawkAuth
+        }
+
+        let rule = helperMap[helperType]
+        if (rule) {
+            return rule(params, helper)
+        }
+
+        const schemeSetupMap = {
+            Basic: ::this._extractBasicAuth,
+            Digest: ::this._extractDigestAuth,
+            OAuth: ::this._extractOAuth1,
```

Comment:
Trying quickly the other day, I couldn't find where they store the OAuth 2 infos…